src/GAP.jl

```diff
@@ -105,8 +122,21 @@ function __init__()
     gap_module = @__MODULE__
     Base.MainInclude.eval(:(__JULIAGAPMODULE = $gap_module))
     if ! isdefined(Main, :__GAPINTERNAL_LOADED_FROM_GAP) || Main.__GAPINTERNAL_LOADED_FROM_GAP != true
-        run_it(GAPROOT, error_handler_func)
+        run_gap(GAPROOT, error_handler_func)
     end
+
+    ## FIXME: Hack because abstract types cannot be endowed with methods in 1.1
+    ## At 1.2, this should be possible and this hack can be replaced with the
+    ## correspponding line in ccalls.jl
```

Comment:
```suggestion
    ## corresponding line in ccalls.jl
```
Though... what *is* the "corresponding line"? Couldn't this be made explicit here, by just saying what the replacement should be...

Reply:
Will do so.